src/MSCorEE.Desktop/MSCorEE.Helpers+IClrMetaHost.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
```

Comment:
Not sure about the file name here as for now we kept `+` to mean nested type. Maybe `MSCorEE.Helpers.IClrMetaHost.cs` ?